style.css

```diff
@@ -398,3 +398,110 @@ span {
   background: #fff;
   margin-bottom: 30px;
 }
+
+.desktop-tools {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .nav-link {
+    display: none;
+  }
+
+  .nav-container {
+    padding: 10px 40px;
+  }
+
+  .desktop-tools {
+    display: inline;
+  }
+
+  .display-tools,
+  li,
+  a {
+    padding-right: 10px;
+    height: 0%;
+  }
+
+  .major-container {
+    padding: 40px 50px;
+  }
+
+  .project-card {
+    display: grid;
+    padding: 10px;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .project-card img {
+    height: 100%;
+    width: 100%;
+  }
+
+  .prject-card-info {
+    padding: 20px;
+  }
+
+  .order-2 {
+    order: 2;
+  }
+
+  .about-container {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .language {
+    display: block;
+  }
+
+  .lang-text {
+    display: flex;
+  }
+
+  .lang-div {
+    flex-direction: row;
+    padding: 5px 15px;
+  }
+
+  .lang-list {
+    padding: 20px;
+    flex-direction: column;
+    width: 122px;
+    height: 120px;
+  }
+
+  .contact-form {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    padding: 5rem 1.25rem 0;
+  }
+
```

Comment:
- Hey, please, let's make some adjustments by adding the correct background image and reducing the width of the text so as for the text not to be too much apart from the inputs. I made a highlight in the images below so you can see how to adjust it. Let's fix this to improve our design as much as we can below:


*Figma design*
![Screenshot from 2022-12-16 17-22-04](https://user-images.githubusercontent.com/59397159/208143977-787c818b-8f70-41f1-95d8-29fb877615ec.png)

*your design*
![Screenshot from 2022-12-16 17-21-24](https://user-images.githubusercontent.com/59397159/208143981-d79842b5-6014-47d1-89cb-16172eaeca25.png)